modules/standard/Random.chpl

```diff
@@ -857,10 +857,13 @@ module Random {
            the interval [`min`, `max`] can be constructed in this way.
 
        */
-      proc getNext(min: eltType, max:eltType): eltType {
+      proc getNext(min: eltType, max:eltType): eltType throws {
```

Comment:
Does it still need to be `throws` now that the added error is a halt?

Reply:
Does `chplspell` detect non-throwing `throws` functions now too? :)

Reply:
Just wait til you see what else is in the version 2 spell checker!

Reply:
It turns out the introduction of `format()` introduced a throwing function to this function body, so I guess you'll have to fix that in `chplspell 2.1`.